docker/helloworld/dockerize

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
tioli: this `dockerize` pattern works well for us in a number of places, and i bet also helps with image size. something to consider, if we want broader support for [automated builds](https://docs.docker.com/docker-hub/builds/#create-an-automated-build), we need to encapsulate the entire image build process within the Dockerfile.